packages/itinerary-body/src/index.js

```diff
@@ -200,7 +204,17 @@ ItineraryBody.propTypes = {
    * - leg: the transit leg
    * - stopsExpanded: whether the intermediate stop display is currently expanded
    */
-  TransitLegSummary: PropTypes.elementType.isRequired
+  TransitLegSummary: PropTypes.elementType.isRequired,
+  /**
+   * A custom SVG icon component inserted into the transit alert toggle button
```

Comment:
The component can be any graphics content, I suppose.

Reply:
you're right, it doesn't have to specifically be an SVG, really any component could be passed here.  I will update these docstrings to reflect that

Reply:
removed the bit about SVG from the docstrings for the new props, I amended my existing commit and force pushed fyi in case you'll have this branch checked out locally